lib/src/compile.test.ts

```diff
@@ -14,9 +14,9 @@ describe('compile', () => {
   describe('success', () => {
     describe('input', () => {
       it('compiles an SCSS string by default', async () => {
-        expect(await compileString({source: 'a {b: c}'})).toBe(
-          'a {\n  b: c;\n}'
-        );
+        expect(await compileString({source: 'a {b: c}'})).toMatchObject({
```

Comment:
`toMatchObject` is too permissive here, let's use the stricter `toEqual` instead. (same for the other tests below)

```suggestion
        expect(await compileString({source: 'a {b: c}'})).toEqual({
```